.gitlab-ci.yml

```diff
@@ -7,13 +7,13 @@ include:
 
 variables:
   SERVICE_NAME: indexd
-  LANGUAGE_VERSION: python3.8
-  BASE_CONTAINER_VERSION: 3.0.7
+  LANGUAGE_VERSION: python3.12
+  BASE_CONTAINER_VERSION: 3.1.0
 
 tox:
   parallel:
     matrix:
-      - LANGUAGE_VERSION: [python3.8, python3.9, python3.10]
+      - LANGUAGE_VERSION: [python3.12, python3.13]
```

Comment:
The PR description mentions updating to Python3.12. Was this and the corresponding entry for py3.13 in the tox.ini meant to be retained for this PR?

Reply:
Yeah, since this is a service instead of a library, I don't think the matrix of tox is needed.  Unless you're trying to future proof it? However, that isn't the way the other service repos are set up.

Reply:
I will bump it to py3.13, this helped me confirm it works in py3.13